cmd/report.go

```diff
@@ -66,21 +54,17 @@ func createReport(execScheduler *execution.Scheduler, importedModules []string,
 		if !builtinOutputsIndex[o] {
 			continue
 		}
-		publicOutputs = append(publicOutputs, o)
+		u.Strings("outputs", o)
```

Comment:
Anyway, you're touching the logic - maybe also clean previous comments related to the go1.20. Maybe as a separated commit?